idea_plugin/build.gradle

```diff
@@ -15,31 +15,35 @@
  */
 
 plugins {
-  id "org.jetbrains.intellij" version "0.7.2"
+  id "org.jetbrains.intellij" version "1.2.1"
 }
 
 repositories {
   mavenCentral()
 }
 
 ext {
-  googleJavaFormatVersion = '1.10.0'
+  googleJavaFormatVersion = "1.12.0"
 }
 
-apply plugin: 'org.jetbrains.intellij'
-apply plugin: 'java'
+apply plugin: "org.jetbrains.intellij"
+apply plugin: "java"
+
+sourceCompatibility = JavaVersion.VERSION_11
+targetCompatibility = JavaVersion.VERSION_11
 
 intellij {
   pluginName = "google-java-format"
-  version = "211.6693.65-EAP-SNAPSHOT"
+  plugins = ["java"]
+  version = "2020.3"
 }
 
 patchPluginXml {
   pluginDescription = "Formats source code using the google-java-format tool. This version of " +
                       "the plugin uses version ${googleJavaFormatVersion} of the tool."
-  version = "${googleJavaFormatVersion}.0"
-  sinceBuild = '201'
-  untilBuild = ''
+  version.set("${googleJavaFormatVersion}.0")
+  sinceBuild = "203"
```

Comment:
Curious why you updated this to 2020.3? AFAIK there's no reason it shouldn't continue to work with 2020.1. (I just tested the 1.13 plugin with 2020.1 and it still seems to work...)

Reply:
The reason I updated it to 2020.3 was for the newer features regarding notification groups
https://plugins.jetbrains.com/docs/intellij/notifications.html#top-level-notifications-balloons
\+ https://plugins.jetbrains.com/docs/marketplace/product-versions-in-use-statistics.html

I considered it acceptable to move to 203 from 201 to get the newer features to ease the pain in the future

Reply:
Oh okay, yeah that's totally reasonable. I thought it was just an arbitrary change, but if it's enabling that notification configuration change, that defintiely makes sense.